conftest.py

```diff
@@ -49,3 +50,5 @@ def pytest_collection_modifyitems(config, items):
             item.add_marker(interpreter_skip)
         if backend_name == 'PlaidML' and 'skip_on_plaidml' in item.keywords:
             item.add_marker(plaidml_skip)
+        if backend_name == 'INTELGPU' and 'skip_on_intelgpu' in item.keywords:
```

Comment:
Doesn't all above conditions exclusive? We should use `elif` here.

Reply:
@postrational can you answer this question?